src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -50,18 +50,24 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DesignTimeAssemblySearchPaths>$(AssemblySearchPaths)</DesignTimeAssemblySearchPaths>
   </PropertyGroup>
 
-  <!-- Temporary hacks to work around bugs -->
+  <!-- Temporary workarounds -->
   <PropertyGroup>
-    <!-- Temp Hack: https://github.com/dotnet/roslyn/issues/12167 -->
+    <!-- Workaround: https://github.com/dotnet/roslyn/issues/12167 -->
     <NoLogo>true</NoLogo>
 
-    <!-- Temp Hack: https://github.com/Microsoft/msbuild/issues/720 -->
+    <!-- Workaround: https://github.com/Microsoft/msbuild/issues/720 -->
     <OverrideToolHost Condition=" '$(DotnetHostPath)' != '' and '$(OverrideToolHost)' == ''">$(DotnetHostPath)</OverrideToolHost>
```

Comment:
I think this can be removed. It should no longer be needed/used.

(Obviously leaving it in now doesn't hurt anything. So maybe we should remove it after RC.)